src/NuGet.Core/NuGet.CommandLine.XPlat/Commands/PackageSearch/JsonFormat/SearchResultPackage.cs

```diff
@@ -0,0 +1,13 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using NuGet.Protocol.Core.Types;
+
+namespace NuGet.CommandLine.XPlat
+{
+    internal class SearchResultPackage
+    {
+        public IPackageSearchMetadata PackageSearchMetadata { get; set; }
+        public string DeprecationMessage { get; set; }
```

Comment:
Isn't DeprecationMEssage a part of PackageSearchMetadata? 
I'm guessing we don't need it here. 

That'd mean that we probably don't need this class anymore.

Reply:
Fixed